hashes/src/hash160.rs

```diff
@@ -49,13 +49,11 @@ fn from_engine(e: HashEngine) -> Hash {
 #[cfg(test)]
 mod tests {
     #[test]
-    #[cfg(feature = "alloc")]
```

Comment:
Thanks for the contribution! These changes are good, this feature guard is stale from when we used `vec`. Could you please pull them into a separate patch (or PR even) and justify the patch/PR with something like "test code is unnecessarily feature gated".